src/Resources/views/BlockAdmin/select_type.html.twig

```diff
@@ -21,21 +21,26 @@ file that was distributed with this source code.
             </h3>
         </div>
         <div class="box-body">
-            {% for code, service in services %}
+            {% for code, service in services if service.metadata is defined or service.blockMetadata is defined %}
+                {% if service.metadata is defined %}
```

Comment:
```suggestion
                {% set metadata = service.metadata|default(service.blockMetadata) %}
```

Reply:
This won't work if one of the methods does not exist

Reply:
Oh, well... I never tested the `default` filter with nonexistent methods.